Target-MVD-Mathias/Networking/Services/UserAPI.swift

```diff
@@ -74,6 +74,27 @@ class UserAPI {
       failure(error)
     }
   }
+  
+  class func signup(name: String, email: String, password: String, gender: String, success: @escaping (_ responseObject: [String: Any]) -> Void, failure: @escaping (_ error: Error) -> Void) {
+    let parameters = [
```

Comment:
This is because in the swift-base project there are 2 spaces for indentation and the default in Xcode is 4 spaces. What number of spaces should I use for indentation?